eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -55,6 +55,11 @@ object TestConstants {
     val mandatory = 50000
   }
 
+  case object LowTestFeature extends Feature with InitFeature with NodeFeature {
```

Comment:
That name can be improved...If we start having multiple unknown test features, we should rename `TestFeature` to `UnknownExperimentalFeature` (IIRC we called that range "experimental range") and `LowTestFeature` to `UnknownSpecFeature` or something like this?

Reply:
The distinction between "spec feature" and "experimental feature" is irrelevant here and would only add confusion, what matters is that one uses a low number and the other one uses a high number which is why I named it like that. I could rename `TestFeature` to `HighTestFeature` to be more consistent.

Reply:
I thought you meant to use that to test the difference between feature bits in the experimental range allowed for blips vs feature bits in the range reserved for bolts (which we currently don't distinguish). This misunderstanding does show that the name is quite bad though.

What's the threshold between low and high number? That's completely arbitrary and not explained, as it depends on _what_ you plan to test with this, right? Can you clarify your goal here? That would help find the right name for this.

Reply:
It was used to test that invoices wouldn't crash with features higher than 5120. I've removed it since we've decided that invoices can't have unknown features.